torch/nn/functional.py

```diff
@@ -2204,6 +2205,11 @@ def mse_loss(input, target, size_average=None, reduce=None, reduction='mean'):
 
     See :class:`~torch.nn.MSELoss` for details.
     """
+    if not (target.size() == input.size()):
```

Comment:
It would be more readable and efficient to do `target.size() != input.size()`. Can we please use that here and above as well?